src/contexts/Sidebar/SidebarActions.js

```diff
@@ -1 +1,14 @@
-//code to be inputted later
+export const EXPAND = "EXPAND";
+export const CLOSE = "CLOSE";
+export const CHANGE = "CHANGE";
+
+//Is this necessary? They're not really imported anywhere from what I can see.
+export const actionExpandSidebar = (category) => ({
+  type: EXPAND,
+  payload: category,
+});
+export const actionCloseSidebar = () => ({ type: CLOSE });
+export const actionChangeSidebar = (category) => ({
+  type: CHANGE,
+  payload: category,
```

Comment:
I think I do prefer the use of `data` vs. `payload` here, although it could be even more semantic with `{type: CHANGE, category: category}`